1.10/deploying-services/containerizers/ucr.md

```diff
@@ -19,7 +19,7 @@ The Universal Container Runtime offers the following advantages:
 ## Prerequisite
 If your service [pulls Docker images from a private registry](/docs/1.10/deploying-services/private-docker-registry/), you must specify the [`cluster_docker_credentials_path` in your `config.yaml`](/docs/1.10/installing/custom/configuration/configuration-parameters/#cluster_docker_credentials) file before installing DC/OS.
 
-1. Specify the UCR from the web interface. Go to **Services**  > **Run a Service** > **Single Container** > **More Settings**. In the **Container Runtime** section, choose the **Universal Container Runtime** radio button.
+1. Specify the UCR from the web interface. Go to **Services**  > **Run a Service** > **Single Container** > **More Settings**. In the **Container Runtime** section, choose the **MESOS RUNTIME** radio button.
 
 1. In the **Container Image** field, enter your container image.
```

Comment:
it is NOT necessary to have a container image to run UCR...   The Mesos container runtime is now UCR... 
A user can choose to use Docker images with the Docker engine under the covers... they can also use docker images with UCR.   AND commands run without a docker image is still UCR.   
It might be easiest to state... that you "may" enter a container image if you like.